### PR TITLE
Publish new release 1.5.0

### PR DESCRIPTION
**Pull request by alexandrekarst**

### CODE REVIEW

File: Example/IAdvizeSwiftExample/IAdvizeSwiftExample/Source/AppDelegate+iAdvize.swift [hunk 1]
```diff
@@ -66,7 +66,7 @@ extension AppDelegate {
         //
         // Your `iAdvizeSecret` is available on your app on the iAdvize Administration website.
         // TODO: replace YOURTARGETINGRULEUUID by your own rule.
-        IAdvizeManager.shared.activate(jwtOption: .secret(iAdvizeSecret), externalId: "ConnectedUserIdentifier", ruleId: UUID(uuidString: "_YOUR_RULE_ID_")!) { success, isEnabled in
+        IAdvizeManager.shared.activate(jwtOption: .secret(iAdvizeSecret), externalId: "freezer@gmail.com", ruleId: UUID(uuidString: "7a45cb38-1864-436e-935d-9720c6aed031")!) { success, isEnabled in
             guard success else {
```
Comment (ycoupe) [inline Example/IAdvizeSwiftExample/IAdvizeSwiftExample/Source/AppDelegate+iAdvize.swift]: Remove moi ces identifiants steup :)

File: Example/IAdvizeSwiftExample/IAdvizeSwiftExample/Source/AppDelegate.swift [hunk 2]
```diff
@@ -14,8 +14,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-    let iAdvizeApplicationID = "_YOUR_APPLICATION_ID_"
-    let iAdvizeSecret = "_YOUR_SECRET_"
+    let iAdvizeApplicationID = "a478675e-af16-4a9c-a833-8d2cba8d0cc9"
+    let iAdvizeSecret = "Nk/DS3YPJQgwWxpIGmAhvrAFerUwGEkA52ay3qf8EHU="
```
Comment (ycoupe) [inline Example/IAdvizeSwiftExample/IAdvizeSwiftExample/Source/AppDelegate.swift]: Same

File: Example/IAdvizeSwiftExample/Podfile.lock [hunk 3]
```diff
@@ -1,42 +1,57 @@
...
-  - MessageKit (2.0.0):
-    - MessageInputBar/Core
-  - SwiftGraylog (1.0.0)
+  - MessageKit (3.0.0):
+    - InputBarAccessoryView (~> 4.2.2)
+  - SwiftGraylog (1.1.0)
 
 DEPENDENCIES:
-  - iAdvize
+  - iAdvize (from `../../../mobile-ios-sdk-conversation-library`)
```
Comment (ycoupe) [inline Example/IAdvizeSwiftExample/Podfile.lock]: Remove this

**Review by ycoupe — APPROVED** [review]
Sinon ok

File: Example/IAdvizeSwiftExample/IAdvizeSwiftExample/Source/AppDelegate+iAdvize.swift (hunk 1 above)
Comment (alexandrekarst) [reply Example/IAdvizeSwiftExample/IAdvizeSwiftExample/Source/AppDelegate+iAdvize.swift]: Haha XD thx

**Pull request merged**